dummy-api/matrix-service-api.yaml

```diff
@@ -0,0 +1,124 @@
+swagger: '2.0'
+basePath: '/v0'
+info:
+  title: HCA Matrix Service
+  version: 0.0.0-dummy
+schemes:
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  api_key:
+    type: apiKey
+    name: x-api-key
+    in: header
+definitions:
+  MatrixRequest:
+    type: object
+    properties:
+      bundle_fqids:
+        type: array
+        items:
+          type: string
+          format: dcp-fqid
+      bundle_fqids_url:
+        type: string
+        format: url
+      filter:
+        $ref: '#/definitions/MatrixFilter'
+      format:
+        $ref: '#/definitions/MatrixFormat'
+  MatrixFilter:
+    type: object
+    properties:
+      pandas_filter_string:
+        type: string
+  MatrixFormat:
+    type: string
+    enum:
```

Comment:
This should be reduced to the set of formats supported in v0. Based on today's discussion that would be:
-zarr
-loom
-csv _which is an addition_